### PR TITLE
Akshaya p

### PR DESCRIPTION
**Pull request by AKSHAYAPALCHAMY-P**

### CODE REVIEW

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Subtraction/Answer.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System;
+namespace Digit
+{
+    class Subtraction
+    {
```
Comment (msindia-learning) [inline AkshayaP/00 - Projects/03 - C#/Problem Practice/Subtraction/Answer.cs]: ### Add naming rules

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+using System;
```
Comment (msindia-learning) [inline AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs]: ### Add naming rules

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+int count = 0;
+
+for(int i = 0; i < mystring.Length - 1; i++)
+{
+    if(mystring[i] == mystring[i + 1])
+    {
+        count++;
+    }
+}
```
Comment (msindia-learning) [inline AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs]: ### Print output - Console.WriteLine(nCount);

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+using System;
+
+string n = Console.ReadLine();
+string mystring = Console.ReadLine();
+
+int count = 0;
+
+for(int i = 0; i < mystring.Length - 1; i++)
```
Comment (msindia-learning) [inline AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs]: ### Alternative way. 
 - mystring.Length - 1 can replaced by n - 1 (should convert n as integer)

**Just for hint**

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Subtraction/Answer.cs (hunk 1 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/03 - C#/Problem Practice/Subtraction/Answer.cs]: Done

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs (hunk 3 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs]: Done

File: AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs (hunk 2 above)
Comment (AKSHAYAPALCHAMY-P) [reply AkshayaP/00 - Projects/03 - C#/Problem Practice/Stone on the Table/Answer.cs]: Done

**Review by msindia-learning — APPROVED** [review]

**Pull request merged**